frontend/src/components/Body/BodyOne/BodyOne.css

```diff
@@ -0,0 +1,63 @@
+.BodyOne {
```

Comment:
a couple general CSS comments that i will leave here instead of in all of your CSS files:

1) good [CSS styling](https://getbem.com/) would dictate that we standardize to hyphen naming across the board. so in this case, let's do `.body-one`
2) in conjunction with 1, CSS classes are generally lower cased everywhere